docs/http-api.md

```diff
@@ -523,9 +523,7 @@ $ curl \
 
 ```json
 {
-  "data": {
-    "key_ids": ["6D0A9151F25B6B85"]
-  }
+  "data": ["6D0A9151F25B6B85"]
```

Comment:
Did you mean to change this? Just an opportunistic update?

Reply:
Oh I see you did update it.

Reply:
Yes and yes, fixed a quick bug while at it, otherwise `vault list` doesn't work